controller/users.go

```diff
@@ -215,9 +215,43 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 func (c *UsersController) List(ctx *app.ListUsersContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
 		var err error
-		var users []*account.User
 		var result *app.UserArray
-		users, err = appl.Users().List(ctx.Context)
+
+		if ctx.Username != nil {
+			// From a data model perspective, we are querying by identity ( and not user )
+			identities, err := appl.Identities().Query(account.IdentityFilterByUsername(*ctx.Username))
```

Comment:
Wouldn't this be a `Show` operation ? I understand username and email will return one user. But if not defined, we list all the users. That is weird part in my opinion.

The only point could be that the same email can be used by multiple users. WDYT?